packages/pageflow/src/editor/views/EditMetaDataView.js

```diff
@@ -30,10 +30,12 @@ export const EditMetaDataView = Marionette.Layout.extend({
     var state = this.options.state || {};
     var features = this.options.features || {};
     var editor = this.options.editor || {};
+    var entryTypeName = (editor.entryType && editor.entryType.name) || 'none';
```

Comment:
`editor.entryType` is always present in "real life". If this is here to make specs work, I'd suggest to instead register an entry type in the affected spec. We already do this in some places, when views depend on entry types.